### PR TITLE
[FIX]: Email Validation

### PR DESCRIPTION
**Pull request by adityad85**
* Added the error for users to know if the email format is wrong by validating it.

### CODE REVIEW

File: src/helpers/validations.js [hunk 1]
```diff
@@ -1,10 +1,15 @@
 const required = value => (value ? undefined : 'Required');
+
 const minValue = min => value => (value.toString().length >= min ? undefined : `Should be greater than ${min}`);
+
+const emailFormat = value => (!value.match(/^([a-zA-Z0-9_.-])+@(([a-zA-Z0-9-])+\.)+([a-zA-Z0-9]{2,4})+$/) ? 'Enter Valid email' : undefined);
```
Comment (anshurathee) [inline src/helpers/validations.js:5]: Did you write it yourself, or just copy ? :P

Comment (adityad85) [reply src/helpers/validations.js:5]: I can explain this regex. :P

File: src/Router/appRoutes.js [hunk 2]
```diff
@@ -15,6 +15,7 @@ const appRoutes = (
         <Route path={routes.ClientLogin} component={LoginPage} />
         <PrivateRoute exact path="/" component={HomePage} />
         <Route path={routes.ClientSignup} component={RegisterPage} />
+        <Route component={LoginPage} />
```
Comment (anshurathee) [inline src/Router/appRoutes.js:18]: What about the path for this component ?

Comment (adityad85) [reply src/Router/appRoutes.js:18]: default route.If the user types some random address

Comment (anshurathee) [reply src/Router/appRoutes.js:18]: okay, I didn't see it wrapped in the switch component

**Review by anshurathee — APPROVED** [review]

**Pull request merged**